command/console.go

```diff
@@ -63,6 +63,9 @@ func (c *ConsoleCommand) RunContext(ctx context.Context, cla *ConsoleArgs) int {
 		return ret
 	}
 
+	diags := c.DetectBundledPlugins(packerStarter)
```

Comment:
So, the reason why I included this template all over is because for HCL2 templates we call `DetectPluginBinaries`, which was in the `Initialize` call prior to this commit.
The reason why I moved it out was because otherwise we'd potentially get multiple errors when running the bundled plugin detection, though I don't remember exactly which error we get.

Reply:
That being said, if initialize and console don't rely on plugins, it doesn't matter if they don't detect existing plugins and we can probably remove that.

Reply:
Tested locally, it does seem to work fine without detecting the plugins, let's remove this change from those two commands then!